include/z64player.h

```diff
@@ -318,6 +395,14 @@ typedef enum PlayerFacialExpression {
     /* 15 */ PLAYER_FACE_15
 } PlayerFacialExpression;
 
+#define GET_GESTURE_FROM_JOINTTABLE(jointTable)           (((PlayerAnimationFrame *)(jointTable))->gestureInfo)
+#define GET_EYE_INDEX_FROM_JOINTTABLE(jointTable)         ((GET_GESTURE_FROM_JOINTTABLE(jointTable) & 0xF) - 1)
+#define GET_MOUTH_INDEX_FROM_JOINTTABLE(jointTable)       (((GET_GESTURE_FROM_JOINTTABLE(jointTable) >> 4) & 0xF) - 1)
```

Comment:
```suggestion
#define GET_GESTURE_FROM_JOINT_TABLE(jointTable)           (((PlayerAnimationFrame *)(jointTable))->gestureInfo)
#define GET_EYE_INDEX_FROM_JOINT_TABLE(jointTable)         ((GET_GESTURE_FROM_JOINTTABLE(jointTable) & 0xF) - 1)
#define GET_MOUTH_INDEX_FROM_JOINT_TABLE(jointTable)       (((GET_GESTURE_FROM_JOINTTABLE(jointTable) >> 4) & 0xF) - 1)
```
I think joint table is two words